src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -266,14 +274,11 @@ protected DefaultDockerClient(final Builder builder) {
     // ApacheConnector doesn't respect per-request timeout settings.
     // Workaround: instead create a client with infinite read timeout,
     // and use it for waitContainer, stopContainer, attachContainer, logs, and build
-    final RequestConfig noReadTimeoutRequestConfig = RequestConfig.copy(requestConfig)
-        .setSocketTimeout((int) NO_TIMEOUT)
-        .build();
-    this.noTimeoutClient = ClientBuilder.newBuilder()
-        .withConfig(config)
+    final RequestConfig noReadTimeoutRequestConfig =
+        RequestConfig.copy(requestConfig).setSocketTimeout((int) NO_TIMEOUT).build();
+    this.noTimeoutClient = ClientBuilder.newBuilder().withConfig(config)
         .property(ApacheClientProperties.CONNECTION_MANAGER, noTimeoutCm)
-        .property(ApacheClientProperties.REQUEST_CONFIG, noReadTimeoutRequestConfig)
-        .build();
+        .property(ApacheClientProperties.REQUEST_CONFIG, noReadTimeoutRequestConfig).build();
```

Comment:
Hm, seems like a lot of these changes are unrelated to the actual network feature. Perhaps we can revert just to make the diff easier to review?